nixos/modules/services/hardware/pid-fan-controller.nix

```
@@ -0,0 +1,163 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.pid-fan-controller;
+  heatSource = {
+    options = {
+      name = lib.mkOption {
+        type = lib.types.uniq lib.types.nonEmptyStr;
+        description = "name of heat source";
```

```suggestion
        description = "Name of the heat source.";
```

Please review the user-facing descriptions for every option and rewrite them with proper grammar and capitalisation. We don't want to show "developer UI" to users.

---

I am not sure if i caught all grammar mistakes, but at least `languagetool` doesn't find any mistakes after my rewording. Are descriptions supposed to end with a period even if they are not proper English sentences (i.e. don't contain a verb)?

---

Yes, as they're still descriptions.